openapi/paths/ready-to-pay.yaml

```
@@ -15,9 +15,9 @@ post:
     intersected with the last matched [Rules Engine](https://user-api-docs.rebilly.com/tag/Rules#operation/PutEventRule)
     `adjust-ready-to-pay` action on `ready-to-pay-requested` event.
 
-    If there were no actions matched for the specific request – all methods supported by the Gateway Accounts are sent.
+    If there were no actions matched for the specific request - all methods supported by the Gateway Accounts are sent.
 
-    To invert this behavior – place an all-matching rule at the very end of the `ready-to-pay-requested` event in
+    To invert this behavior - place an all-matching rule at the very end of the `ready-to-pay-requested` event in
```

```suggestion
    To invert this behavior place an all-matching rule at the very end of the `ready-to-pay-requested` event in
```
I think we don't use "all-matching" in our API or interface?

---

By this I've tried to say
> Create a rule without conditions

We do display `This rule matches all cases.` text inside our interface for such cases.